### ISSUE TITLE
How to specify pod cidr and service cidr?

### ISSUE THREAD

**Issue by wangzhilin90** [labring/sealos#1764]
Previous versions of sealos could specify pod and service networkcidr through  sealos init -- podcidr -- servicecidr.
But for  sealos version 4.0, there is no option to specify these two network 

the following is the SealOS command
Usage:
  sealos run [flags]

Examples:

create cluster to your baremetal server, appoint the iplist:
        sealos run labring/kubernetes:v1.24.0 --masters 192.168.0.2,192.168.0.3,192.168.0.4 \
                --nodes 192.168.0.5,192.168.0.6,192.168.0.7 --passwd xxx
  multi image:
    sealos run labring/kubernetes:v1.24.0 calico:v3.22.1 \
        --masters 192.168.64.2,192.168.64.22,192.168.64.20 --nodes 192.168.64.21,192.168.64.19

**Comment by fanux** [comment]
https://www.sealos.io/docs/getting-started/customize-cluster

**Comment by wangzhilin90** [comment]
> https://www.sealos.io/docs/getting-started/customize-cluster

it's useful to specify pod network，but i want specify service network, is there any idea?thanks

**Comment by willzhang** [comment]
maybe same method
```
networking:
  serviceSubnet: 10.96.0.0/12
  podSubnet: 192.168.0.0/16
```

**Comment by wangzhilin90** [comment]
> maybe same method
> 
> ```
> networking:
>   serviceSubnet: 10.96.0.0/12
>   podSubnet: 192.168.0.0/16
> ```


i have tried ,but not work

**Comment by 5codes** [comment]
```
networking:
  serviceSubnet: "100.55.0.0/16"
  podSubnet: "55.55.0.0/16"
  dnsDomain: "cluster.local"
apiServer:
  certSANs:
    - "100.55.0.1"
  timeoutForControlPlane: 4m0s
```
I have tried like this,the Cluster cloud be created.when I tried to add --master,I got a message as followed.
`error execution phase control-plane-prepare/certs: error creating PKI assets: failed to write or validate certificate "apiserver": certificate apiserver is invalid: x509: certificate is valid for 127.0.0.1, 192.168.26.6, 192.168.26.4, 10.96.0.1, 10.103.97.2, 192.168.26.5, not 100.55.0.1`

**Comment by fanux** [comment]
Provide docs about how to add certSANs

**Comment by cuisongliu** [comment]
https://github.com/labring-actions/bug-verify/actions/runs/4161615545

**Comment by cuisongliu** [comment]
new version is fixed 4.1.5-rc1 test is ok.
